go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/losisin/helm-values-schema-json/v2
 go 1.24.2
 
 require (
+	github.com/fxamacker/cbor/v2 v2.8.0
```

Comment:
Note on this: I wanted the cache to use some form of binary encoding, as encoding JSON as a JSON string is very inefficient as it requires a lot of escaping. I first went with `encoding/gob` but your Bearer security scanner told me it was too insecure. (apparently gob doesn't validate things like field types and would try to load a string into a `time.Time` and weird stuff like that)

I searched around for a bit and settled on CBOR, which seems fast and secure enough for our purposes.